api_tests/registrations/views/test_registration_detail.py

```diff
@@ -684,8 +691,9 @@ def test_read_write_contributor_cannot_update_admin_writeable_fields(
                 ]
             }
         }
+        del payload['data']['attributes']  # just institutions
```

Comment:
This test was broken and only testing for the attributes in the payload, but it would have worked if written properly.